src/templates/en/2021/methodology.html

```diff
@@ -0,0 +1,474 @@
+{% extends "base/methodology.html" %}
+
+{% block title %}Methodology | The Web Almanac by HTTP Archive{% endblock %}
+
+{% block description %}Describes how the {{ year }} Web Almanac was put together: The Datasets and Tools used and how the project was run.{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac methodology{% endblock %}
+
+{% block index %}
+      <ul>
+        <li><a href="#overview">Overview</a></li>
+        <li><a href="#dataset">About the dataset</a>
+          <ul>
+            <li><a href="#websites">Websites</a></li>
+            <li><a href="#metrics">Metrics</a></li>
+          </ul>
+        </li>
+        <li><a href="#tools">Tools</a>
+          <ul>
+            <li><a href="#webpagetest">WebPageTest</a></li>
+            <li><a href="#lighthouse">Lighthouse</a></li>
+            <li><a href="#wappalyzer">Wappalyzer</a></li>
+            <li><a href="#chrome-ux-report">Chrome UX Report</a></li>
```

Comment:
Should we add a section on Blink data?

CC: @VictorLeP

Reply:
Added